src/main/java/net/starlark/java/eval/StarlarkSemantics.java

```diff
@@ -174,6 +174,9 @@ public Builder setBool(String name, boolean value) {
 
     /** Returns an immutable StarlarkSemantics. */
     public StarlarkSemantics build() {
+      if (map.isEmpty()) {
```

Comment:
Nit: It looks like this just micro-optimizes away the few lines it would take `copyOf()` to allocate an iterator and realize that the map is empty. But constructing a `StarlarkSemantics` doesn't happen very often. Delete?

Reply:
It's also meant to make equality checks (such as in `net.starlark.java.eval.CallUtils#getStarlarkClassDescriptor`) between "default" semantics instances as fast as possible. It's still a micro-optimization, but it's not just relevant for construction.